### ISSUE TITLE
VS Codium 1.66.1 RPM has wrong dependencies and won't install in RHEL 7

### ISSUE THREAD

**Issue by ajkerzner** [VSCodium/vscodium#1043]
**Describe the bug**
Seems to basically be https://github.com/microsoft/vscode/issues/146390, which was introduced in VS Code 1.66.0 and fixed in 1.66.1. This bug is still present in VS Codium 1.66.1, though.

**Please confirm that this problem is VSCodium-specific**
- [ X] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [ X] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**Desktop (please complete the following information):**
 - RHEL 7.9

**Comment by paulcarroty** [comment]
Fixed in VSC insiders, will be available in next release.

**Comment by ajkerzner** [comment]
@paulcarroty This still appears to be an issue with 1.66.2.

**Comment by ajkerzner** [comment]
Can this bug be reopened? I admit I don't have enough experience with this tooling to fix this myself.
Affected version: 1.66.0, 1.66.1, 1.66.2
VS Code had the same issue in 1.66.0, but it was fixed by 1.66.1, so as of 1.66.1 this bug is now specific to VS Codium.

**Comment by ajkerzner** [comment]
@paulcarroty Problem still present in vscodium 1.67. It was fixed VS Code in 1.66.1. Request reopen.

**Comment by daiyam** [comment]
Not sure if it's related but I had to disabled a new flag (`FAIL_BUILD_FOR_NEW_DEPENDENCIES`) so that linux releases can be built. (https://github.com/VSCodium/vscodium/pull/1066/files#diff-e39cd79316b4da9e7c1e409f2cb11654eba3d7d1840191a0c19f1d0fda95a332)
I will to try to look into it.
VSCode has moved to an Alpine based docker image while we are still using the Debian based image.

**Comment by ajkerzner** [comment]
I don't know if this is helpful.
Old: https://github.com/VSCodium/vscodium/runs/5419614524?check_suite_focus=true#step:13:822
New: https://github.com/VSCodium/vscodium/runs/6334517686?check_suite_focus=true#step:13:1168

I wonder if the new dependency thing works for VS Code's build system but not ours.

**Comment by ajkerzner** [comment]
Do we need to go back to a prior ubuntu platform again?

Possibly related: #242

**Comment by ajkerzner** [comment]
@daiyam Actually I think it's related. I think that flag is actually meant to catch this.


This is a list for testing: https://github.com/microsoft/vscode/blob/main/build/linux/rpm/dep-lists.js
This is the thing that generates a list: https://github.com/microsoft/vscode/blob/main/build/linux/rpm/dependencies-generator.js#L56

So I think this test failing _may_ have been the first indication of a problem - the dependencies in the codium build exceed those expected in the test.

**Comment by daiyam** [comment]
> I wonder if the new dependency thing works for VS Code's build system but not ours.

I tried but I can't access their logs.

This check is new so I assumed it was an extra check and the diffs were due to Alpine/Debian.

> So I think this test failing _may_ have been the first indication of a problem - the dependencies in the codium build exceed those expected in the test.

I agree but I wanted to get this release out because of the git extension/vsxopen issue which was rendering codium nearly useless...
And they were some much problems in this build.
The 32bit versions are out yet because the dependencies aren't correctly installing...

This week-end, I will look into it.
But feel free to make a PR ;)
